build_container.sh

```diff
@@ -51,6 +51,9 @@ cd -
 # can modify the Dockerfile to use whatever REPO is in the environment.
 TAG=$DISTRO_TO_BUILD-builder
 dockerdir=`find -name $TAG`
+# use the builder template to populate the distro specific Dockerfile
+cp dockerfiles/templates/Dockerfile.builder $dockerdir/Dockerfile
+sed -i "s/DISTRO_TO_BUILD/$DISTRO_TO_BUILD/g" $dockerdir/Dockerfile
```

Comment:
Why the sed & cp, why not
`sed "s/DISTRO_TO_BUILD/$DISTRO_TO_BUILD/g" dockerfiles/templates/Dockerfile.builder > $dockerdir/Dockerfile`

Probably not worth changing the patch just for this.